### PR TITLE
Pix gateway

### PR DESCRIPTION
**Pull request by netojoaobatista**
Add PIX as payment available to Brazil

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ sudo: required
 language: php
 
 php:
-  - 5.6
+  - 7.3
```
Comment (jvmoreira) [inline .travis.yml:7]: Will this impact compatibility with merchants that are currently using our WooCommerce gateway with PHP <7.3 versions?

File: tests/bin/test.sh [hunk 2]
```diff
@@ -22,6 +22,6 @@ setup_docker() {
   docker-compose up -d
 }
 
-setup_docker
-
-while ! curl -s http://localhost > /dev/null; do echo waiting for woocommerce-container; sleep 10; done; run_tests
+# setup_docker
+#
+# while ! curl -s http://localhost > /dev/null; do echo waiting for woocommerce-container; sleep 10; done; run_tests
```
Comment (jvmoreira) [inline tests/bin/test.sh:27]: Why were these tests disabled?

File: lib/Plugin/Services/WC_EBANX_Payment_Adapter.php [hunk 3]
```diff
@@ -195,6 +195,12 @@ private static function get_phone_number( $order, $configs, $gateway_id ) {
 	private static function get_document( $configs, $names, $gateway_id ) {
 		$country = trim( strtolower( WC()->customer->get_billing_country() ) );
 
+		if ('no' === $configs->settings['require_documents']) {
+			return '';
+		}
+
+		error_log(print_r($configs, true));
```
Comment (jvmoreira) [inline lib/Plugin/Services/WC_EBANX_Payment_Adapter.php:202]: Is this `error_log` supposed to be here?

File: .travis.yml (hunk 1 above)
Comment (netojoaobatista) [reply .travis.yml:7]: Those who are using the plugin NOW will not be impacted. But by now, all merchants who want to use this plugin will have to met the plugin requirements; those requirements are the same as the [benjamin](https://github.com/ebanx/benjamin).

**Pull request closed without merging**